metakb/database.py

```diff
@@ -203,6 +203,15 @@ def _add_descriptor(tx, descriptor: Dict, added_ids: Set[str]):
                                                       'description', 'xrefs',
                                                       'alternate_labels'))
 
+        if descr_type == 'TherapyDescriptor':
+            # handle extensions field in therapy descriptor
```

Comment:
I think a note here that we're grabbing approval for evidence assessment in particular would be more helpful